### ISSUE TITLE
sims_maf dependency on ipython-notebook obsolete?

### ISSUE THREAD

**Issue by rhiannonlynne** [mjuric/conda-lsst#74]
So sims_maf has a dependency on ipython-notebook >= 3.0 
(line 104/105 https://github.com/mjuric/conda-lsst/blob/master/etc/config.yaml)

but I believe with jupyter notebook, that the 'ipython-notebook' package is deprecated. 

I don't know if we require a specific version of jupyter or ipython, but if it's already a recent enough version, we should just drop the ipython-notebook requirement.  I'm not entirely sure where the change happened, but I think it was with ipython 4.0.

**Comment by mjuric** [comment]
We added the requirement because people were confused when they installed `sims_maf` and didn't have IPython :).

What would you prefer -- to have them install `(ipython|jupyter)-notebook` manually, or just bump up the requirement to `jupyter-notebook`?

**Comment by rhiannonlynne** [comment]
Is there a jupyter-notebook? (I don't see one if I do conda search jupyter-notebook). 
If they have jupyter_core > 4.0, I think that they have jupyter notebook automatically, or anyway, I can't figure out which of the jupyter modules (jupyter, jupyter_core, jupyter_client or jupyter_console) it comes with.

**Comment by rhiannonlynne** [comment]
Does jupyter come with miniconda? If so, I think the easiest thing to do is drop any requirement. If people don't have jupyter or ipython notebook capabilities (because they have a very old version of anaconda/miniconda), they can install it manually. And upgrade conda.

**Comment by mjuric** [comment]
No, miniconda comes with nothing; that's why we added the requirement.

## Fine with me to drop it, but you'll have to update the docs to make sure people know to install it on their own.

**Comment by rhiannonlynne** [comment]
Hmm. Maybe we can add the relevant 'jupyter' to the dependencies?
